docs/v2/examples/get-month-year.md

```diff
@@ -8,8 +8,8 @@ Get month or year.
 const calendar = new HelloWeek({
     selector: '.calendar',
     onNavigation: () => {
-        console.log('Current Month: ', calendar.getMonth());
-        console.log('Current Year: ', calendar.getYear());
+        console.log('Current Month: ', calendar.date.getMonth());
+        console.log('Current Year: ', calendar.date.getYear());
```

Comment:
In fact the correct is that:
```js
console.log('Current Month: ', calendar.getMonth());	        
console.log('Current Year: ', calendar.getYear());
```
because we are accessing to a public method from HelloWeek.

Reply:
Hmm,
Not sure but it didn't work.
I'll close this.